### ISSUE TITLE
libfplll broken

### ISSUE THREAD

**Issue by olexandr-konovalov** [gap-system/gap-docker-base#10]
When trying to load Float in the new GAP 4.9.1 `gap-docker` container, I have
```
gap> LoadPackage("float");
#W dlopen() error: libfplll.so.5: cannot open shared object file: No such file or dire\
ctory
Error, module '/home/gap/inst/gap-4.9.1/pkg/float-0.9.0/bin/x86_64-pc-linux-gnu-default64/flo\
at.so' not found in
  if not LOAD_DYN( arg[1], false ) then
    Error( "no support for dynamic loading" );
fi; at /home/gap/inst/gap-4.9.1/lib/files.gd:583 called from 
LoadDynamicModule( f 
 ); at /home/gap/inst/gap-4.9.1/pkg/float-0.9.0/init.g:43 called from
<function "unknown">( <arguments> )
 called from read-eval loop at /home/gap/inst/gap-4.9.1/pkg/float-0.9.0/init.g:45
Error, was not in any namespace at /home/gap/inst/gap-4.9.1/lib/variable.g:291 called from
LEAVE_NAMESPACE(  ); at /home/gap/inst/gap-4.9.1/lib/package.gi:1210 called from
ReadPackage( pkgname, "init.g" 
 ); at /home/gap/inst/gap-4.9.1/lib/package.gi:1583 called from
<function "LoadPackage">( <arguments> )
 called from read-eval loop at *stdin*:1
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
brk>
```

**Comment by olexandr-konovalov** [comment]
Now works!

**Comment by sebasguts** [comment]
Yep, was fixed via 3af75eb32bbd314b16342caab9fcf0e5a9489af7 (which I forgot to push earlier, sorry)
